pages/platform/kubernetes-k8s/known-limits/guide.en-asia.md

```diff
@@ -50,12 +50,20 @@ There is also a limit of 10 open ports on every `LoadBalancer`, and these ports
 
 Our Managed Kubernetes service is based on OpenStack, and your nodes and persistent volumes are built on it, using OVH Public Cloud. As such, you can see them in the *Servers* section of [OVH Cloud Manager](https://www.ovh.com/manager/cloud/). It doesn't mean that you can deal directly with these nodes and persistent volumes as other cloud instances. The *managed* part of OVHcloud Managed Kubernetes means that we have configured those nodes and volumes to be part of our Managed Kubernetes. Please refrain from manipulating them from the *OVH Cloud Manager* (modifying ports left opened, renaming, resizing volumes...), as you could break them.
 
-## Ports and Private Networks
+## Ports
 
 In any case, there are some ports that you shouldn't block on your instances if you want to keep your OVHcloud Managed Kubernetes service running:
 
-- Port 22 (*ssh*)
-- Port 10250 (*kubelet*)
-- Ports from 30000 to 32767 (*NodePort* services port range)
+### Ports to open from internet
```

Comment:
I agree that "the internet" is not clear enough.
What do you think of "Ports to open from public network" instead of "Ports to open from all sources", I worry that "source" is not clear enough neither?

Reply:
Indeed, `Public network` sounds better than `sources`, I love this one :)

Reply:
Agreed! In the ovh context, someone interested by this part of the doc has a good chance of looking for information to configure openstack security groups (that was my case), where it makes sense to talk about public network.
I suggested "sources" because in a security group rule definition, the wording used is src & dest.
So both work but public network is probably more inclusive :)